app/src/main/java/com/example/myapplication/AuthActivity.kt

```diff
@@ -21,37 +23,43 @@ class AuthActivity : AppCompatActivity() {
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
         setContentView(R.layout.activity_auth)
```

Comment:
1) in "onCreate" should be only function. Decomposition please
   2) Maybe use binding?
  3)  autologin NOT implemented, fix this please